### ISSUE TITLE
Add a discord chat

### ISSUE THREAD

**Issue by therealadityashankar** [subins2000/p2pt#7]
there are so-so many things possible from this,
- one can build a truly decentralized social network !, that's actually peer to peer hosted
- If one build a wasm/javascript blockchain, one could probably build entire e-commerce/video-sharing/practically possibly anything with this

I didn't know this was possible, I was initially quite frustrated by the nature of webtorrent to only provide static sites, but this is truly amazing !

I'd love to talk more about this, but this is like really freaking awesome

**Comment by subins2000** [comment]
Yes, the possibilities are endless. We can use the WebTorrent tracker standard, and build more apps with it, making WebTorrent a good ecosystem for both torrent and apps !

The initial idea was to build a decentralized proxy for Wikipedia, [it works](https://github.com/subins2000/p2wiki), but the docs are not complete and we have a plan to build it better with [wikivue](https://github.com/santhoshtr/wikivue).

I've started a [discord server](https://discord.gg/7TE4uD)

**Comment by Weedshaker** [comment]
Hello,
do you have a new Discord Link? The one above expired...

I got some familiar question to this topic: https://github.com/subins2000/p2pt/issues/49

Thanks a lot!

**Comment by subins2000** [comment]
Can't keep up with a discord, WebTorrent discord is good enough, p2pt specific discussions can be here in github itself
